src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String NAME_OF_OPERATION_SUPPLY = "supply";
+    private static final String NAME_OF_OPERATION_BUY = "buy";
+    private static final int INDEX_OF_OPERATION_TYPE = 0;
+    private static final int INDEX_OF_AMOUNT = 1;
+    private static final int INDEX_OF_SUPPLY_AMOUNT = 0;
+    private static final int INDEX_OF_BUY_AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] operationsAmounts = readAmountsOfOperations(fromFileName);
+        int supplyAmount = operationsAmounts[INDEX_OF_SUPPLY_AMOUNT];
+        int buyAmount = operationsAmounts[INDEX_OF_BUY_AMOUNT];
+
+        String report = createReportMessage(supplyAmount, buyAmount);
+        writeReportToFile(report, toFileName);
+    }
+
+    private static int[] readAmountsOfOperations(String fromFileName) {
```

Comment:
```suggestion
    private int[] readAmountsOfOperations(String fromFileName) {
```

same for other methods